src/features/query-bar/messages.js

```diff
@@ -7,6 +7,16 @@ const messages = defineMessages({
         description: 'Text displayed on the Tooltip for an input field',
         id: 'boxui.queryBar.tooltipEnterValueError',
     },
+    tooltipInvalidFloatError: {
+        defaultMessage: 'Please Enter a Float',
```

Comment:
Not sure if "Float" is a useful message for all of our users. Maybe "Decimal number"?